syntax = "proto3";

service Showtime {
    rpc GetListShowtimes(Empty) returns (stream Schedule) {}
    rpc GetMoviebyDate(Date) returns (Schedule) {}
}

message Schedule {
    string date = 1;
    repeated string movies = 2;
}

message Date {
    string date = 1;
}

message Empty {
}